lib/ansible/cli/__init__.py

```diff
@@ -422,6 +422,12 @@ def post_process_args(self, options):
                     skip_tags.add(tag.strip())
             options.skip_tags = list(skip_tags)
 
+        # Make sure path argument doesn't have a backslash
+        if hasattr(options, 'action') and (options.action == 'install' or options.action == 'download'):
+            if hasattr(options, 'args') and len(options.args) > 0:
+                path = options.args[0]
+                options.args = path.rstrip("/")
```

Comment:
Since `options.args` is a list, you'll need to strip each item in the list:

```suggestion
                options.args = [path.rstrip("/") for path in options.args]
```